src/style.css

```diff
@@ -0,0 +1,109 @@
+body {
+  background-color: #ffff;
+  margin: 0;
+}
+
+.container {
+  display: flex;
+  flex-flow: row;
+  flex-wrap: wrap;
+  justify-content: center;
+}
+
+.box {
+  height: 300px;
+  min-width: 33.33%;
+  text-align: center;
+  font-size: 100px;
+  line-height: 300px;
+  color: #ffff;
+  font-family: Arial, serif;
+}
+
+.box:nth-child(1) {
+  background: rgb(255, 0, 0);
+  order: 1;
+  flex-grow: 1;
+}
+
+.box:nth-child(2) {
+  background: rgb(204, 0, 0);
+  order: 2;
+  flex-grow: 1;
+}
+
+.box:nth-child(3) {
+  background: rgb(153, 0, 0);
+  order: 3;
+  flex-grow: 1;
+}
+
+.box:nth-child(4) {
+  background: rgb(102, 0, 0);
+  order: 6;
+  flex-grow: 1;
+}
+
+.box:nth-child(5) {
+  background: rgb(51, 0, 0);
+  order: 5;
+  flex-grow: 1;
+}
+
+.box:nth-child(6) {
+  background: rgb(0, 0, 0);
+  order: 4;
+  flex-grow: 1;
```

Comment:
this property is similar to all boxes, move it to `.box`, so you don't need to write it for each box